aula_5/Exercício1/Exercicio1.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Exercício 1 - Fácil - Slide 4
+ * Escreva um programa que guarde seu nome em uma variável chamada `nome`.
+ * Execute e veja o resultado no console.
+ * 
+ * Extra:
+ * [] faça o programa receber o nome como entrada
+ * [] permita que o usuário insira seu nome completo,mas exiba apenas o primeiro nome
+ * dica: descubra a diferença entre next() e nextline()
```

Comment:
Como o `next` e `nextLine` não são "funções globais", acho legal mencionar que são métodos da classe `Scanner`. 

Pra quem tá iniciando, talvez nem entenda 100% esses conceitos, mas acho interessante já ir usando as terminologias mais precisas.

Julgo que usar uma terminologia precisa aqui não vá causar desentendimento.

```suggestion
 * dica: descubra a diferença entre os métodos next() e nextLine() da classe Scanner
```